### PR TITLE
Fix up flight_replay.py doc/example to DKPY2. Note, still has bug #355

### PR DESCRIPTION
**Pull request by hamishwillee**
This fixes up the flight_replay example and associated docs to be "DKPY2 compliant". This includes adding vehicle.close and connect, and making the example docs follow the same pattern as the other example docs.

NOTE:

* This does not work! There is an exception due to #355 . The docs have a note to this effect. When the bug is fixed we'll have to revisit and retest the docs/code.
* This hamishwillee dronecode key and api. The old version failed because droneshare now insists on a key with the correct format.
* Even though this does not "work", it is a better example to have for DKPY2 for people to learn from. The old example would not work either due to the bad key!

### CODE REVIEW

File: docs/examples/flight_replay.rst [hunk 1]
```diff
@@ -2,57 +2,95 @@
...
+    explains how to set up *Mission Planner* to view a vehicle running on the simulator (SITL).
+
+On the command prompt you should see (something like):
+
+.. code-block:: bash
+
+
+    \dronekit-python\examples\flight_replay>flight_replay.py
+    Connecting to vehicle on: 127.0.0.1:14550
+    >>> ☺APM:Copter V3.4-dev (e0810c2e)
```
Comment (tcr3dr) [inline docs/examples/flight_replay.rst]: Can you remove these smiley faces? They're just a bug on master :) Was an errant, random character.

File: examples/flight_replay/flight_replay.py [hunk 2]
```diff
@@ -63,7 +71,11 @@ def download_messages(mission_id, max_freq = 1.0):
 def replay_mission(payload):
     """Given mission JSON, set a series of wpts approximating the previous flight"""
     # Pull out just the global position msgs
-    messages = payload['messages']
+    try:
+        messages = payload['messages']
+    except:
+        print "Exception: payload from site is: %s" % payload
+        exit()
```
Comment (tcr3dr) [inline examples/flight_replay/flight_replay.py]: Should this be `sys.exit`?

File: examples/flight_replay/flight_replay.py [hunk 3]
```diff
@@ -94,17 +106,20 @@ def replay_mission(payload):
...
+max_freq = 0.1
+json = download_messages(mission_id, max_freq)
+print "JSON downloaded..."
+replay_mission(json)
+
+
+#Close vehicle object before exiting script
+print "Close vehicle object"
+vehicle.close()
+print("Completed")
```
Comment (tcr3dr) [inline examples/flight_replay/flight_replay.py]: No parens.

File: examples/flight_replay/flight_replay.py (hunk 2 above)
Comment (hamishwillee) [inline examples/flight_replay/flight_replay.py]: Yes. Wonder why exit() does actually work?

**Comment by hamishwillee** [comment]
Fixed. Ready to merge.

**Pull request merged**